### ISSUE TITLE
Remote Audio 

### ISSUE THREAD

**Issue by davxy** [any1/wayvnc#209]
Is there a way to listen to audio coming to remote applications? If I'm not wrong currently only video output can be shared.

If not currently possible, is this planned?

**Comment by any1** [comment]
Yeah, this is planned, but it looks like I didn't make an issue for it.

The main hurdles are the following:
 * There does not exist a protocol extension for compressed audio. We would have to make one and advocate for its adoption.
 * Because framebuffer updates may be sent as large continuous blobs, audio packets may be stalled for the duration of sending a framebuffer update. This means that audio playback needs to be delayed for the maximum transfer duration of such an update for smooth playback.
 * Some applications require audio and video to be synchronised. We've added timestamps to video frames, but it's an undocumented extension that's only supported by `wlvncc` currently. We would also have to advocate the adoption of video frame timestamps. Strict A/V sync is optional though. Could be added later.

**Comment by soreau** [comment]
For what it's worth, as a workaround for pulse (or pipewire-pulse) users on the LAN, you can use RTP multicast like so:

Server:
`pactl load-module module-null-sink sink_name=rtp`
`pactl load-module module-rtp-send source=rtp.monitor`
`pactl set-default-sink rtp`

Clients:
`pactl load-module module-rtp-recv`

**Comment by heroin-moose** [comment]
Maybe it's possible to start with QEMU Audio and extending it with the support for compressed audio?
